DEVELOPMENT.md

```diff
@@ -0,0 +1,19 @@
+# Development environment
+
+In order to get started developing the extension, you need a postgres install
+with timescaledb, and Rust dependencies. For more information, see the
+[compile from source](INSTALL.md#Compile From Source) install instructions.
+
+To spare you the effort of getting this set up yourself, we provide a docker
+image with all required dependencies, which allows you to just get started.
+
+Run `make devenv` to build the docker image, start it, and expose it on port
+54321 on your local machine. This docker image mounts the current directory
+into the `/code` directory in the container. It runs postgres 14 and
+continually recompiles and reinstalls the promscale extension on source
+modifications. This means that you can edit the sources locally, and run tests
```

Comment:
It would be nice to provide commands for running test. Something like: 
`POSTGRES_URL=postgres://ubuntu@localhost:54321/ make test`? I actually can't find make target to just run tests

Reply:
➕

Reply:
Yes, that would be in the scope of what Valeriy is working on.

Reply:
@sumerman I don't think that is possible, because make runs every command in a subshell, so the `export` is swallowed.

Reply:
Yeah, that sounds good!